stdlib/runtime/numbers.gr

```diff
@@ -0,0 +1,1336 @@
+/* compilation-mode: runtime */
```

Comment:
To be frank, my eyes glossed over while reviewing this file so I don't trust I would have actually caught anything.